nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -403,6 +403,26 @@
      installer after creating <literal>/var/lib/nextcloud</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>/etc/sysctl.d/nixos.conf</literal> file containing all the options set via
+      <link linkend="opt-boot.kernel.sysctl">boot.kernel.sysctl</link> was moved to
+      <literal>/etc/sysctl.d/60-nixos.conf</literal>, as <literal>sysctl.d(5)</literal>
+      recommends prefixing all filenames in <literal>/etc/sysctl.d</literal> with a
+      two-digit number and a dash to simplify the ordering of the files.
+    </para>
+   </listitem>
+   <listitem>
+     <para>
+       We now install the sysctl snippets shipped with systemd. Apart from some loose reverse path
+       filtering and source routing filtering, this enables <literal>fq_codel</literal> as a packet
+       scheduler to fight bufferbloat, and configures <literal>systemd-coredump</literal> properly.
```

Comment:
I maybe should have left the above comment down a few lines. In general when it comes to release notes and setting changes, avoid [weasel words](https://en.wikipedia.org/wiki/Weasel_word).

Reply:
About the rp_filter option from above comment: systemd sets `net.ipv4.conf.all.rp_filter` to `2` by default, which does enable some reverse path filtering, while we currently don't do any reverse path filtering at all. On hardened and google-compute-config, we set it to `1` (strict).

Detailled explanation about the options:
 - 0 No source address validation is performed and any packet is forwarded to the destination network
 - 1 Strict Mode as defined in RFC 3074. Each incoming packet to a router is tested against the routing table and if the interface that the packet is received on is not the best return path for the packet then the packet is dropped.
 - 2 Loose mode as defines in RFC 3074 Loose Reverse Path. Each incoming packet is tested against the route table and the packet is dropped if the source address is not routable through any interface. The allows for asymmetric routing where the return path may not be the same as the source path.

As a lot of distros enabled the systemd sysctls, they enabled loose reverse path filtering, so I thought it should be fine to do so too. Same goes for `net.ipv4.conf.all.accept_source_route = 0`.

I didn't want to explicitly list every option currently set in `${systemd}/example/sysctl.d/50-default.conf` - only the fact that we now apply these options, plus some broad summary about what's changed, so people are aware when reading release notes.